### DISCUSSION TITLE
Feature Request: 独立Web服务、桌面客户端

### DISCUSSION THREAD

**Discussion by AiurArtanis** [Quorafind/Obsidian-Thino#584] [Polls 投票]
### Feature Requested | 请求的功能

参照Flomo，web端+客户端+OB内如果都能全部打通和同步，这样的话整个生态就比较完善了

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

**Comment by juestchaos** [comment]
独立 web 版已经在内测了，成为社区优质贡献者，会提前拿到内测全
